packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -162,6 +149,27 @@ export class W3mAccountDefaultWidget extends LitElement {
     `
   }
 
+  private activityTemplate() {
+    const isSolana =
+      ChainController.state.activeChain === ConstantsUtil.CHAIN_NAME.SOLANA.toLocaleLowerCase()
```

Comment:
We shouldn't need to use `toLocaleLowerCase()`, instead you could use `.CHAIN.SOLANA`